src/connectedvmware/azext_connectedvmware/custom.py

```diff
@@ -1611,7 +1609,7 @@ def enable_guest_agent(
     vm_client = cf_virtual_machine(cmd.cli_ctx)
 
     if is_system_identity_enabled(vm_client, resource_group_name, vm_name) is False:
-        enable_system_identity(vm_client, resource_group_name, vm_name, no_wait)
+        _ = enable_system_identity(vm_client, resource_group_name, vm_name, False).result()
```

Comment:
> /mnt/vss/_work/1/s/src/connectedvmware/azext_connectedvmware/custom.py:1612:9: F841 local variable '_' is assigned to but never used
1     F841 local variable '_' is assigned to but never used

Please fix the style issue:

```suggestion
        enable_system_identity(vm_client, resource_group_name, vm_name, False).result()
```